llvm/test/CodeGen/SPIRV/struct_opaque_pointers.ll

```diff
@@ -0,0 +1,15 @@
+; RUN: llc -O0 -mtriple=spirv32-unknown-unknown %s -o - | FileCheck %s
+
```

Comment:
Could spirv-val run line be added here as well?

Reply:
As in the case of https://github.com/llvm/llvm-project/pull/78293, adding spirv-val run line would make the test fail. The context is that both SPIR-V Backend and SPIR-V Translator do not comply with specification in the part that the constituents of OpConstantComposite must all be non-specialization constant-instruction declarations or OpUndef, with the difference that spirv-val tolerates SPIR-V Translator's wrapping of operands in OpSpecConstantComposite.
I'd suggest to postpone this to be a separate issue if it's acceptable, as in the case of #78293.